app/assets/stylesheets/govuk_publishing_components/components/_chevron-banner.scss

```diff
@@ -0,0 +1,125 @@
+$red: #e61e32;
+$dark-red: #b31424;
+$yellow: #ffdd00;
+
+.gem-c-chevron-banner {
+  position: relative;
+}
+
+.gem-c-chevron-banner:hover {
+  .gem-c-chevron-banner__link-container {
+    background-color: $dark-red;
+  }
+
+  .gem-c-chevron-banner__link-container:after {
+    background-image: image-url("govuk_publishing_components/chevron-banner/chevron-banner-hover.svg");
+  }
+}
+
+.gem-c-chevron-banner--hover-border:hover {
+  .gem-c-chevron-banner__link-container {
+    border-right: 2px solid;
+    border-color: govuk-colour("white");
+
+    @include govuk-media-query($from: mobile) {
+      border-right: 0;
+    }
+  }
+
+  .gem-c-chevron-banner__link-container:after {
+    background-image: image-url("govuk_publishing_components/chevron-banner/chevron-banner-hover-border.svg");
+  }
+}
+
+.gem-c-chevron-banner:focus-within .gem-c-chevron-banner__link-container {
+  border-right: 2px solid;
+  border-color: govuk-colour("black");
+  background-color: $yellow;
+
+  @include govuk-media-query($from: mobile) {
+    border-right: 0;
+  }
+}
+
+.gem-c-chevron-banner:focus-within .gem-c-chevron-banner__link-container:after {
+  background-image: image-url("govuk_publishing_components/chevron-banner/chevron-banner-small-focus.svg");
+
+  @include govuk-media-query($from: mobile) {
+    background-image: image-url("govuk_publishing_components/chevron-banner/chevron-banner-focus.svg");
+  }
+}
+
+.gem-c-chevron-banner__link-container {
+  background-color: $red;
+  padding: govuk-spacing(4);
+  border-radius: 5px;
+  border: 2px solid transparent;
+  border-right: 0;
+
+  @include govuk-media-query($from: mobile) {
+    border-radius: 5px 3px 3px 5px;
+    margin-right: 104px;
```

Comment:
There's a very slight edge problem with this:

<img width="125" alt="Screen Shot 2019-09-05 at 10 40 35" src="https://user-images.githubusercontent.com/861310/64331002-efcbd580-cfc9-11e9-912d-5d0e3747bf96.png">

Is the join between the left edge of the chevron and the rest of the red box meant to have slightly rounded corners? If so I'd get an adjusted SVG to include those corners, if not you can shift the background position of the chevron from `-1px` to `0` and remove the border radius on the red box, would look like this:

<img width="142" alt="Screen Shot 2019-09-05 at 10 44 17" src="https://user-images.githubusercontent.com/861310/64331237-581ab700-cfca-11e9-8108-ef0ed79eee83.png">

Reply:
It's not intentional, it's a workaround because if I keep the straight corners the hover and focus state with outlines looks odd. I've asked Mia for a version of the SVGs with those corners included to make it easier 🙂

Reply:
@andysellick I think this has been resolved now with the new SVGs - would you mind taking another look?

Reply:
That seems to have solved it, great.